docs/api/hardhat/getting-started.md

```diff
@@ -2,242 +2,299 @@
+The `zksolc` block contains the minimal configuration required for the compiler:
 
 ```typescript
-networks: {
-        goerli: {
-          url: "https://goerli.infura.io/v3/<API_KEY>", // The Ethereum Web3 RPC URL.
-          zksync: false, // Set to false to target other networks.
-        },
-        zkSyncTestnet: {
-          url: "https://testnet.era.zksync.dev", // The testnet RPC URL of zkSync Era network.
-          ethNetwork: "goerli", // The identifier of the network (e.g. `mainnet` or `goerli`)
-        zksync: true, // Set to true to target zkSync Era.
-        }
-    },
-
+zksolc: {
+  version: "1.3.8",
+  compilerSource: "binary",
+  settings: {},
+},
 ```
 
-::: tip
+
+::: tip Advanced configuration
 To learn more about each specific property in the `hardhat.config.ts` file, check out the [plugins documentation](./plugins.md)
 :::
 
-## Write and deploy a contract
+## Environment variables
 
-1. Create the `contracts` and `deploy` folders. In the `contracts` folder we will store all the smart contract files. In the `deploy` folder we'll place all the scripts related to deploying the contracts.
+This project uses the `dotenv` package to load your private key, which is required to deploy and interact with smart contracts.
 
-2. Create the `contracts/Greeter.sol` contract and paste the following code:
+To configure it, duplicate the `.env.example` file, rename it to `.env`, and enter your wallet private key in it. The `.env` is included in the `.gitignore` file so it won't be uploaded to a repository.
 
-```solidity
-//SPDX-License-Identifier: Unlicensed
-pragma solidity ^0.8.13;
-
-contract Greeter {
-    string private greeting;
+```text
+WALLET_PRIVATE_KEY=abcdef12345....
+```
 
-    constructor(string memory _greeting) {
-        greeting = _greeting;
-    }
+## Compile and deploy a contract
 
-    function greet() public view returns (string memory) {
-        return greeting;
-    }
+Smart contracts are placed in the `contracts` folder. The template project contains a simple `Greeter.sol` contract and a script to deploy it.
 
-    function setGreeting(string memory _greeting) public {
-        greeting = _greeting;
-    }
-}
+To compile the contract, run:
+```sh
+yarn hardhat compile
+``` 
+You'll see the following output:
+```text
+Compiling 1 Solidity file
+Successfully compiled 1 Solidity file
+✨  Done in 1.09s.
 ```
-
-3. Run `yarn hardhat compile` which uses the `hardhat-zksync-solc` plugin to compile the contract. The `artifacts-zk` and `cache-zk` folders will be created in the root directory (instead of the regular Hardhat's `artifacts` and `cache`).
+The `artifacts-zk` and `cache-zk` folders will be created in the root directory (instead of the regular Hardhat's `artifacts` and `cache`). These folders contain the compilation artifacts (including contract's ABIs) and compiler cache files.
```

Comment:
```suggestion

The `artifacts-zk` and `cache-zk` folders appear in the root directory (instead of the regular Hardhat's `artifacts` and `cache`). These folders contain the compilation artifacts (including contract's ABIs) and compiler cache files.
```